### DISCUSSION TITLE
Problem Installing 2.4 on Proxmox

### DISCUSSION THREAD

**Discussion by bumfighter** [Security-Onion-Solutions/securityonion#12200] [2.4]
### Version

2.4.3

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

6

### RAM

16

### Storage for /

500

### Storage for /nsm

900

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I've tried installing Security Onion multiple times from different versions of 2.4 for the last few months on my proxmox server. Every time I've tried the installation always fails. I reboot and so-status will show everything green but there will never be any network traffic logs.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by reyesj2** [comment]
> Every time I've tried the installation always fails.

Can you provide any logs or the error given when the installation fails? ```/root/errors.log``` and ```/root/sosetup.log``` are helpful to troubleshoot

**Comment by bumfighter** [comment]
![errors log](https://github.com/Security-Onion-Solutions/securityonion/assets/30623646/3ffbbffb-9a81-4b15-962a-38cf3f5b7c0e)
![sosetup log](https://github.com/Security-Onion-Solutions/securityonion/assets/30623646/600791f2-72af-40e8-b0cc-848c48d98ef4)
Here are the screenshots of both of those logs. Doesn't really give much to go off of it seems.

**Reply by reyesj2** [reply]
so-status shows all services are green, but you still have no logs in SOC? There is button on the grid page that allows you to test traffic ingest on your sensor. https://docs.securityonion.net/en/2.4/grid.html#icons-in-lower-left-corner

**Reply by bumfighter** [reply]
![Testpcap](https://github.com/Security-Onion-Solutions/securityonion/assets/30623646/80477c41-2725-4b90-b5e4-11eab7b499c3)
This is the error I get but if I switch to the hunt interface it looks like I do have some logs populating with the standalone as the source http. I noticed the sniffing interface I have for SO has an IP assigned for some reason. Would this be causing a potential issue?

Also, I just did a tcpdump on that same sniffing interface via the standalone cli and it shows all the correct traffic but not logs get created in SOC.

**Reply by reyesj2** [reply]
Your sniffing interface should not have an IP assigned. You should remove it and run ```sudo so-monitor-add``` on the sensor and make sure it is actually assigned as a sniffing interface.

**Comment by bumfighter** [comment]
![so-monitor-add](https://github.com/Security-Onion-Solutions/securityonion/assets/30623646/b00b9509-0418-4fca-8cd9-213464796ead)
I deleted the ip assigned to the interface and performed so-monitor-add to the same interface and the above is what I get. Does the "MTU greater than maximum" error have a solution?

**Reply by bumfighter** [reply]
I say this because there aren't any network logs being produced in kibana or anywhere else.

**Comment by isaacgolding** [comment]
I had a similar issue with proxmox early on.   The issue might be that your dont have promisc turned on in the /etc/network/interfaces file    see this discussion here https://github.com/Security-Onion-Solutions/securityonion/discussions/8245
